picard/util/astrcmp.c

```diff
@@ -170,8 +170,17 @@ static PyMethodDef AstrcmpMethods[] = {
 	{NULL, NULL, 0, NULL}
 };
 
-PyMODINIT_FUNC
-initastrcmp(void)
+static struct PyModuleDef AstrcmpModule =
 {
-	(void)Py_InitModule("astrcmp", AstrcmpMethods);
-}
+    PyModuleDef_HEAD_INIT,
+    "astrcmp", /* name of module */
```

Comment:
Nitpick: either let all comments here begin one space after the comma or align the slashes.